### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - New Cards] - Ensures elements with ARIA roles have all required ARIA attributes

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8358]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 22H2 (OS Build 25300.1000)
URL: https://adaptivecards.io/samples/ActivityUpdate.html
Browser: Edge Dev Version 111.0.1661.36 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to 'Designer' tab item and press enter key to activate it.
3. Navigate to the 'New Card' button and press enter key to activate it.
4. Run accessibility insight for web tool and observe the issue.

**Actual Result:**
Under the 'New Card' screen, Elements with ARIA roles does not have all required ARIA attributes.

**Expected Result:**
Ensures elements with ARIA roles should have all required ARIA attributes so that screen reader convey the proper information about the laebls.

**Element path:** 
div[aria-labelledby="acd-open-sample-item-title-96"] > .acd-open-sample-item-thumbnail > .ac-adaptiveCard.ac-container > .ac-textBlock[role="heading"]

**How to fix:** 
Fix any of the following:
  Required ARIA attribute not present: aria-level

**Note:**
Same issue is repro under 'Activity Update' screen for the for all headings like Input.Text element and others.

**User Impact:**
It will impact the screen reader user if the headings do not have required attributes.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

![MAS4 1 2- Ensures elements with ARIA roles have all required ARIA Attributes](https://user-images.githubusercontent.com/95913748/224284508-1fd5bb3f-29b0-4b26-95e3-15fbd1f1dd9e.png)

![MAS4 1 2- Ensures elements with ARIA roles have all required ARIA Attribute](https://user-images.githubusercontent.com/95913748/224284555-76685d96-deb5-4922-8c7a-567631237ade.png)

![image](https://user-images.githubusercontent.com/95913748/224322731-98627c77-c78f-4098-8bc8-cdf548df1d38.png)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is still repro, hence reactivating this issue.

**Test Environment:**
OS: Version 23H2 (OS Build 25381.1)
URL: https://adaptivecards.io/designer/
Browser: Edge Dev Version 115.0.1880.3 (Official build) dev (64-bit)
Tools: Accessibility Insight For Web

**Snippet:**
![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/2090780d-b16c-47a6-a5e9-f22b874b9d4c)

**Comment by anna-dingler** [comment]
Hi @vagpt,

We just made a release of our website, and this change should now be available. Thanks!

**Comment by vagpt** [comment]
This issue is not repro on the environment below, hence closing this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 23H2(OS Build 25931.1000)
Browser Version: 118.0.2060.1

**Snippet:**

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/d58552e7-8305-461a-82a8-a2661dcd584e)
